consul/kv_consul.go

```diff
@@ -840,6 +840,14 @@ func (kv *consulKV) EnumerateWithSelect(
 	return nil, kvdb.ErrNotSupported
 }
 
+func (kv *consulKV) EnumerateKVPsWithSelect(
```

Comment:
should it be EnumerateKvpWithSelect instead ?

Reply:
done